Clicker/Models/Course.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  Course.swift
+//  Clicker
+//
+//  Created by Kevin Chan on 9/26/17.
+//  Copyright © 2017 CornellAppDev. All rights reserved.
+//
+
+import UIKit
+
+enum SubjectType {
+    // Will add rest later if we plan
+    // on keeping this
+    case CS, INFO, MATH
+}
+
+class Course: NSObject {
```

Comment:
Good work, but what's reasoning for making them NSObject?

Reply:
Oh it was because for the user I wanted to keep track of a user's courses in a Set so conforming to NSObject would make it Hashable. For some reason I assumed we aren't displaying the courses of a user. If we actually want to display the courses a user has in like a TableView, though, storing the courses as an array would probably make more sense.

Reply:
I think it would be fine to put them in arrays. When we display them in table views we'll probably copy the arrays inside view controllers

Reply:
Alright, I'll change it then recommit